### PR TITLE
core: add MeanStdDev() method to Mat

### PR DESCRIPTION
**Pull request by morfeo8marc**

### CODE REVIEW

**Comment by deadprogram** [comment]
Hello @morfeo8marc thank you for the PR. Could you please rebase against the latest `dev` branch to resolve the merge conflicts?

File: core.go [hunk 1]
```diff
@@ -594,6 +594,15 @@ func DFT(src Mat, dst Mat) {
 	C.Mat_DFT(src.p, dst.p)
 }
 
+// Calculates a mean and standard deviation of array elements.
```
Comment (willdurand) [inline core.go]: In Go, comments start with the identifier, e.g., "MeanStdDev calculates a mean..."

**Comment by berak** [comment]
don't forget to run: 

    gofmt -w path/to/core.go
    gofmt -w path/to/core_test.go

to make the build bot's happy, just saying ;)

**Comment by berak** [comment]
@morfeo8marc  oh no ;(

don't give up so easily !

there would have been like "all the time in the world", to solve your issues ! (there's no hurry)

**Comment by willdurand** [comment]
I second @berak and we could help you @morfeo8marc

**Pull request closed without merging**